forum/profiles/urls.py

```diff
@@ -13,4 +13,7 @@
 router_startup_save = SimpleRouter()
 router_startup_save.register('startups', SaveStartupViewSet, basename='startups')
 
```

Comment:
Using different routes (e.g. DefaultRouter for basic and SimpleRouter for specific scenarios) increases flexibility.
In this code, SimpleRouter doesn't provide a clear advantage. Unless you need special paths, it's better to stick with one type of router.

router = DefaultRouter()
router.register('investor-profile', InvestorViewSet, basename='investor-profile')
router.register('startup-profile', StartupProfileViewSet, basename='startup-profile')
router.register('startups', SaveStartupViewSet, basename='startups')
router.register('public-startups', PublicStartupViewSet, basename='public-startups')

Reply:
done